pkg/agent/manager/cache/jwt_cache_test.go

```diff
@@ -221,6 +222,47 @@ func TestJWTSVIDCache(t *testing.T) {
 	}
 }
 
+func TestJWTSVIDCacheSize(t *testing.T) {
+	fakeMetrics := fakemetrics.New()
+	log, _ := test.NewNullLogger()
+	log.Level = logrus.DebugLevel
+	cache := NewJWTSVIDCache(log, fakeMetrics, 2)
+
+	now := time.Now()
+	jwtSvid1 := &client.JWTSVID{Token: "1", IssuedAt: now, ExpiresAt: now.Add(time.Second)}
```

Comment:
A one second validity period feels too aggressive and prone to test flakiness if later on we start evicting cache entries periodically based on expiration. I see this validity period is already being used in other tests in this file, but would you mind using a slightly longer validity period like one minute for at least these new tests?